test/test_prototype_transforms.py

```diff
@@ -152,3 +159,32 @@ def test_normalize(self, transform, input):
     )
     def test_random_resized_crop(self, transform, input):
         transform(input)
+
+    @parametrize(
+        [
+            (
+                transforms.ConvertImageColorSpace(color_space=new_color_space, old_color_space=old_color_space),
+                itertools.chain.from_iterable(
+                    [
+                        fn(color_spaces=[old_color_space])
+                        for fn in (
+                            make_images,
+                            make_vanilla_tensor_images,
+                            make_pil_images,
+                        )
+                    ]
+                ),
+            )
+            for old_color_space, new_color_space in itertools.product(
+                [
+                    features.ColorSpace.GRAYSCALE,
+                    features.ColorSpace.GRAYSCALE_ALPHA,
+                    features.ColorSpace.RGB,
+                    features.ColorSpace.RGBA,
+                ],
+                repeat=2,
+            )
+        ]
+    )
```

Comment:
Nit: It is not only related to this one but for previous as well. IMO these parametrizations are really hard to read and understand...  If we could simplify that somehow, it would be nice. Major difficulty is get what is passed to `input` for example...

Reply:
It is indeed hard to read. Refactoring the tests is at the top of my todo list.